documentation/diagnostics-client-library-instructions.md

```diff
@@ -224,14 +224,36 @@ public static void PrintEventsLive(int processId)
 
 This sample shows how to attach an ICorProfiler to a process (profiler attach).
 ```cs
-public static int AttachProfiler(int processId, Guid profilerGuid, string profilerPath)
+public static void AttachProfiler(int processId, Guid profilerGuid, string profilerPath)
 {
     var client = new DiagnosticsClient(processId);
-    return client.AttachProfiler(TimeSpan.FromSeconds(10), profilerGuid, profilerPath);
+    client.AttachProfiler(TimeSpan.FromSeconds(10), profilerGuid, profilerPath);
 }
 ```
 
+#### 8. Set an ICorProfiler to be used as the startup profiler
 
+This sample shows how to request that the runtime use an ICorProfiler as the startup profiler (not as an attaching profiler). It is only valid to issue this command while the runtime is paused in "reverse server" mode.
```

Comment:
I think the reversed server is orthogonal, the runtime just needs to be paused at startup? That startup pause can be accomplished regardless whether the port is listening or connecting:

```
export DOTNET_DiagnosticPorts=foo.sock,connect,suspend
OR
export DOTNET_DiagnosticPorts=foo.sock,listen,suspend
```